.github/workflows/installation.yml

```
@@ -0,0 +1,47 @@
+name: Installation Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

I think you need to set the fetch depth to `0` (infinite) to make sure the history gets picked up for `git tag` to work